### PR TITLE
prep for the release of v2.2.0

### PR DESCRIPTION
**Pull request by StephenCoady**
Signed-off-by: Stephen Coady <scoady@redhat.com>

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -2,6 +2,16 @@
...
 
+v2.2.0
+======
+
+Features
+--------
+
+* Add support for Fedora's new auth backend. This patch adds the ability
+  to interact with fasjson to build the cache of users.
+  (`#310 <https://github.com/fedora-infra/fmn/pull/310>`_).
```
Comment (pypingou) [inline CHANGELOG.rst:13]: There are a few more things to add to the changelog now :)

File: CHANGELOG.rst [hunk 2]
```diff
@@ -2,6 +2,18 @@
...
+* Port the test suite to py3 only and make it pass
+  (https://github.com/fedora-infra/fmn/pull/317)
+
```
Comment (pypingou) [inline CHANGELOG.rst:20]: there are also two changes related to anitya

Comment (StephenCoady) [reply CHANGELOG.rst:20]: woops

**Comment by pypingou** [comment]
Could you maybe squash the commits? That should also re-trigger the tests

**Pull request closed without merging**